addons/website_forum/models/forum.py

```diff
@@ -172,7 +172,7 @@ def _compute_relevancy(self):
     # vote
     vote_ids = fields.One2many('forum.post.vote', 'post_id', string='Votes')
     user_vote = fields.Integer('My Vote', compute='_get_user_vote')
-    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True)
+    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True, default=0)
```

Comment:
@jsh-odoo Make sure that `vote_count` is computed and store at record creation, and that `_get_vote_count` computes correctly `0` in that case.